individual/services.py

```diff
@@ -111,18 +115,19 @@ def create_accept_enrolment_task(self, individual_queryset, benefit_plan_id):
         pass
 
     def _update_json_ext(self, obj_data):
-        if not obj_data or 'json_ext' not in obj_data:
+        if not obj_data or 'json_ext' not in obj_data or 'location_id' not in obj_data:
             return
 
         json_ext = obj_data['json_ext']
         if not json_ext:
             return
 
-        for field in ('first_name', 'last_name', 'dob'):
-            individual_field_value = obj_data.get(field)
-            json_ext_value = json_ext.get(field)
-            if json_ext_value and json_ext_value != individual_field_value:
-                json_ext[field] = individual_field_value
```

Comment:
not sure why this is removed ?

Reply:
It's redundant code where we update in json fields which are a part of basic fields saved directly on db level as db column, not in json_ext. If i remember well, in one of prev PRs or on discord there was a question about that code.

Reply:
@delcroip this is now taken care of by `_align_json_ext` in core as part of https://github.com/openimis/openimis-be-core_py/pull/359. Added tests in this PR shows that the functionality remains the same.